src/redux/reducers/walkthroughServiceReducers.js

```diff
@@ -84,6 +93,31 @@ const walkthroughServiceReducers = (state = initialState, action) => {
           initialState
         }
       );
+    case REJECTED_ACTION(walkthroughTypes.GET_WALKTHROUGH_INFO):
+      return setStateProp(
+        'walkthroughInfo',
+        {
+          error: action.error,
+          errorMessage: action.payload.message
+        },
+        {
+          state,
+          initialState
+        }
+      );
+    case FULFILLED_ACTION(walkthroughTypes.GET_WALKTHROUGH_INFO):
+      return setStateProp(
+        'walkthroughInfo',
+        {
+          pending: false,
+          fulfilled: true,
+          gitUrl: action.payload.data.walkthroughLocations[0].remote,
```

Comment:
It is possible that the `walkthroughLocations` is empty here. If the `WALKTHROUGH_LOCATION` set is a local dir instead of a repo URL it won't currently be included in the list.

At the moment, if I run `yarn start:dev` the app will crash here because of that.

Also, at the moment, the `/about` endpoint returns info about all `WALKTHROUGH_LOCATIONS` added, should be always be taking element 0 here?

If required, we can add a `/about/walkthrough/:walkthroughId` endpoint that returns only info about this walkthrough? If that would be better.